src/test/ui/print_type_sizes/anonymous.rs

```diff
@@ -14,7 +14,10 @@
 // that one cannot control the sizes of these types with the same sort
```

Comment:
Why did you make this change? Seems like purely churn.

Reply:
That was done by @nikomatsakis to fix the tests.

Reply:
In particular, the new definition of `start` based around `Termination` winds up pulling in a lot of code related to panics and unwinds, causing the output of `-Zprint-type-sizes` to have a lot of unrelated gunk in it.